pilot/pkg/serviceregistry/kube/controller/network.go

```diff
@@ -246,12 +245,12 @@ func (c *Controller) getGatewayDetails(svc *model.Service) []model.NetworkGatewa
 	// label based gateways
 	// TODO label based gateways could support being the gateway for multiple networks
 	if nw := svc.Attributes.Labels[label.TopologyNetwork.Name]; nw != "" {
-		if gwPortStr := svc.Attributes.Labels[annotation.NetworkingGatewayPort.Name]; gwPortStr != "" {
+		if gwPortStr := svc.Attributes.Labels[label.NetworkingGatewayPort.Name]; gwPortStr != "" {
```

Comment:
This change is not related to tunneling. I had to change this line, because `NetworkingGatewayPort` was moved to another package.